src/settlements/settlements.controller.ts

```diff
@@ -133,6 +137,36 @@ export class SettlementsController {
     return handleServiceResult(result);
   }
 
+  @ApiTags('instructions')
+  @ApiOperation({
+    summary: 'Reschedule a failed Instruction',
+    description: 'This endpoint will reschedule a failed Instruction',
+  })
+  @ApiParam({
+    name: 'id',
+    description: 'The ID of the Instruction to be rescheduled',
+    type: 'string',
+    example: '123',
+  })
+  @ApiOkResponse({
+    description: 'Details of the transaction',
+    type: TransactionQueueModel,
+  })
+  @ApiTransactionFailedResponse({
```

Comment:
What happens if you try to reschedule an instruction you aren't involved in?

Reply:
Is there Not Found, or is it just Unprocessable?

Reply:
NOT_FOUND will also be there for case where instruction for given ID is not present. `findInstruction` in the service throws that error